docs(rxLocalStorage): Page objects to jsdoc3

Droogans opened the pull request:
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

CITguy commented on src/quarks/rxLocalStorage/scripts/rxLocalStorage.page.js:
```diff
@@ -1,7 +1,20 @@
+     * @param {String} key - The key to store the item under inside of local storage.
+     * @param {String} value - The value to associate with the given `key` under local storage.
+     * @example
+     * rxLocalStorage.setItem('key', 'value');
```
Should these be `encore.rxLocalStorage.*` instead of `rxLocalStorage.*`?

Droogans replied:
`encore`.

I'll fix it.

jj4th commented on src/quarks/rxLocalStorage/scripts/rxLocalStorage.page.js:
```diff
@@ -38,4 +68,6 @@ exports.rxLocalStorage = Page.create({
-});
+};
+
+exports.rxLocalStorage = Page.create(rxLocalStorage);
```
@Droogans elsewhere you've just set this object directly, why the deviating pattern here?  Not that I mind, I actually like this better.

Droogans replied:
This is more a question for @tomreece, who authored this.

It may work best to change the first `var rxLocalStorage = {` to `exports.rxLocalStorage = {`. I'm pretty sure this is a collection of static functions that don't need to be wrapped in `Page.create`.

jj4th commented:
QE-LGTM pending the addressing of Ryan's comments.

Droogans commented:
@CITguy I added additional documentation and tests when I went back and reviewed this module.

I wasn't sure what would happen if you tried to get an item from local storage that did not exist, for documentation purposes. So I added a test for it and documented it.

CITguy commented:
LGTM
